app/worker/tasks/test_exporter.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+
+from django.core.serializers import serialize
+from django.test import TestCase
+
+from app.models.donation import Donation
+from app.models.donor import Donor
+from app.models.item import Item
+from app.models.item_device import ItemDevice
+from app.worker.tasks.exporter import exporter
+
+
+class ExporterTestCase(TestCase):
+    def setUp(self) -> None:
+        donor = Donor.objects.create(
+            donor_name="Test", contact_name="Best", email="test@example.com",
+            mobile_number="+1 (234) 567-8901")
+        donation = Donation.objects.create(donor=donor)
+        item_device = ItemDevice.objects.create()
+        self.item = Item.objects.create(donation=donation,
+                                        device=item_device, quantity=1,
+                                        working=True)
+        return super().setUp()
+
+    def test_exporter(self) -> None:
+        file_name = "test_exporter.csv"
+        queryset = Item.objects.all()
+        qs = serialize(format="json", queryset=queryset)
+        total_count = len(queryset)
+
+        response = exporter(file_name=file_name, qs=qs,
+                            total_count=total_count)
+
+        self.assertEqual(first=response.status_code, second=200)
```

Comment:
nit: check content type to be "application/csv"?

Reply:
Added in https://github.com/CodeTheChangeUBC/reBOOT/pull/344/commits/10827d1928a3ca6f5734ef9e94da474a68f55775.